target/linux/ramips/mt7621/base-files/etc/board.d/02_network

```diff
@@ -199,6 +199,11 @@ ramips_setup_macs()
 	netgear,r6850)
 		wan_mac=$(macaddr_add "$(cat /sys/class/net/eth0/address)" 2)
 		;;
+	wevo,11acnas|\
+	wevo,w2914ns-v2)
+		wan_mac=$(mtd_get_mac_binary factory 0x2e)
+		label_mac=$wan_mac
```

Comment:
I have just added a patch to master that sets lan+1 in an explicit case:
https://github.com/openwrt/openwrt/blob/master/target/linux/ramips/mt7621/base-files/etc/board.d/02_network#L167

So, please rebase your patch onto that and _remove the wevo entries from this new node_.

Reply:
I will merge this patch (patch 2) and rebase myself.